app/helpers/application_helper.rb

```diff
@@ -192,6 +192,11 @@ def auto_complete_search(name, val, options = {})
     text_field_tag(name, val, options)
   end
 
+  def sort(field, permitted: [], **kwargs)
+    kwargs[:url_options] ||= params.permit(permitted + [:locale, :search])
+    super(field, kwargs)
```

Comment:
`super` is always the overridden method from scoped_search as dependencies will be loaded before Foreman's own helpers. I can change it to a chain or prepend, but it seems unnecessary at the moment.